packages/flutter_tools/lib/src/flutter_cache.dart

```diff
@@ -387,12 +386,17 @@ class AndroidGenSnapshotArtifacts extends EngineCachedArtifact {
 /// A cached artifact containing the Maven dependencies used to build Android projects.
 ///
 /// This is a no-op if the android SDK is not available.
```

Comment:
Is this comment still true?

Reply:
The comment appears to be true given https://github.com/flutter/flutter/blob/e8f5a51f79fc5dd2f8de38b4d9f504d09dca8197/packages/flutter_tools/lib/src/flutter_cache.dart#L411-L413, though I am not sure if it _should_. I've created https://github.com/flutter/flutter/issues/127848 to track as a follow-up and added a TODO comment linking to it